src/Components/LandingPage/LandingPage.js

```diff
@@ -1,12 +1,19 @@
-import React, { useState } from 'react'
+import React, { useState, useEffect } from 'react'
 import './LandingPage.css'
 import searchIcon from '../../assets/search.png'
 import sun from '../../assets/sun.svg'
 import locationIcon from '../../assets/location.svg'
 
-function LandingPage() {
-  const [currentLocation, setCurrentLocation] = useState('')
-  const [chooseLocation, setChooseLocation] = useState('')
+function LandingPage({grabUserLocationData, grabInputLocationData}) {
+  const [inputLocation, setInputLocation] = useState('')
```

Comment:
So this is dynamically set up to hold state when a user either inputs a location or uses current location? Nice.